src/pages/PeoplePage/PersonListPage/PersonListPage.tsx

```diff
@@ -1,27 +1,21 @@
 import { IonContent, IonHeader, IonPage } from '@ionic/react'
 import { ReactElement, useState } from 'react'
-import { useQuery } from 'react-query'
 
 import { PeopleMenuBar } from '../../../components/PeopleMenuBar'
-import { getProcessDefinitions } from '../../../lib/queries/getProcessDefinitions'
+import { PersonList } from '../../../components/PersonList'
+import { useAuth } from '../../../lib/useAuth'
 
 export function PersonListPage(): ReactElement {
+  const { user } = useAuth()
```

Comment:
this should be handled in the component?

Reply:
I was trying to extract it out to this page so we wouldn't need to mock Auth in so many places..